pkg/dao/repository_configs.go

```diff
@@ -11,7 +11,7 @@ import (
 	ce "github.com/content-services/content-sources-backend/pkg/errors"
 	"github.com/content-services/content-sources-backend/pkg/models"
 	"github.com/content-services/yummy/pkg/yum"
-	"github.com/jackc/pgconn"
+	"github.com/jackc/pgx/v5/pgconn"
```

Comment:
this I changed because the newer pgx version (v5) uses this package, instead of the independent repository used by pgx v4. It will cause an error in our tests if you use the previous import.